### ISSUE TITLE
Application Hide (not Window Minimize) Function Missing on macos

### ISSUE THREAD

**Issue by flyingray** [docker/for-mac#6446]
- [x] I have tried with the latest version of Docker Desktop
  - [x] I have tried disabling enabled experimental features
  - [x] I have uploaded Diagnostics
  - Diagnostics ID:

### Expected behavior

Nearly all Mac Apps have a menu item "Hide <app name>" and have a key binding Command-h associate with it. It would appear above the Quit menu option.

There is a minimize window button (yellow round button with a dash) but this is to "minimize" not "hide" the app.

This should be a very tiny development effort but has a big user experience benefit.

### Actual behavior

The expected menu and the keybinding are missing.

### Information

  - macOS Version:  any macOS version but using macOS 12.5
  - Intel chip or Apple chip: Intel
  - Docker Desktop Version: v4.11.1

### Steps to reproduce the behavior

  1. Open Docker Desktop app
  2. Try to hide the app either via menu or keybinding

**Comment by nicks** [comment]
maybe a naive question: in Docker Desktop's case, what's the difference between hiding and closing the window? Unlike other mac apps, Docker Desktop stays running when you've closed all the windows, so I'm not sure.

or is it mainly about the key binding, and you'd be fine if command-H closed the window?

**Comment by flyingray** [comment]
@nicks 
Semantics are important but they also extend with scripting and automation beyond just user experience.

Anything wrong with just doing it like every other app in the Mac ecosystem?

**Comment by Quiketembleke** [comment]
https://github.com/docker/for-mac/issues/6446#issue-1335214651https://github.com/docker/for-mac/issues/6446#issuecomment-1211387703

**Comment by cbirdsong** [comment]
When an app is hidden you can very quickly bring it back exactly how you left it, but when you close Docker and reopen it loses all state and always starts on the default screen. Not at all equivalent.

(also, I wonder if fixing this would solve #5962)

**Comment by SethMilliken** [comment]
Please fix this. There have been similar issues for ages that keep getting auto-closed, e.g. https://github.com/docker/for-mac/issues/5813

**Comment by jakubsrnka** [comment]
Just wanted to open an issue for this. Please add this functionality, it's just basically frustrating not to have it there. Thanks!

**Comment by marc-guenther** [comment]
And now #5813 has been closed... :(

I guess this is what happens, when developers neither use nor care about their product. This would be a 10min change for anyone familiar with the source code. I even would invest the time to fix this, as I'm annoyed by it on a daily basis. But of course, this is closed source, so there we go...

**Comment by SethMilliken** [comment]
Glad to see the "acknowledged" tag on this. Since previous related tickets were auto-closed, I'd like to explicitly call attention here to other keybindings and menu items that are also missing from the Docker Desktop app:
```
Preferences     Command-,
Restart         Command-r
Quit            Command-q
```
These were added to the Docker menu widget where they are almost completely useless. These bindings should be added to the Docker Desktop application.

The application is also missing standard Window menu with (at least):
```
Minimize        Command-m
Zoom
```
Ideally, it would also include bindings to navigate through the sidebar items (i.e. Containers, Images, Volumes, Dev Environments, Extensions, etc.).

The app is also missing the standard Help menu.

The "About" menu under "Docker Desktop" menu should be "About Docker Desktop" (again, this is correct in the menu widget, but not the app for some reason).

**Comment by nicks** [comment]
Ah, this shipped a while ago, in Docker Desktop 4.13
